stable/search-prod/templates/cm-search-redis-ca-crt.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
+  name: search-redis-ca-crt
+  namespace: open-cluster-management
+  selfLink: /api/v1/namespaces/open-cluster-management/configmaps/search-redis-ca-crt
```

Comment:
1. Remove `namespace`, while open-cluster-management is our default, we need to support deploying this chart in any other namespace.

2. Remove `selfLink`, this field is auto-generated by kubernetes.